### PR TITLE
Fix of sv6 export

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Still testing this.
#15136

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
doesnt appear to resolve it quite correctly. Will keep investigating.

File: src/openrct2/rct2/S6Exporter.cpp [hunk 1]
```diff
@@ -1123,21 +1143,13 @@ void S6Exporter::RebuildEntityLinks()
...
-                previous = entity->sprite_index;
-            }
-            if (previous != SPRITE_INDEX_NULL)
-            {
-                _s6.sprites[previous].unknown.next_in_quadrant = SPRITE_INDEX_NULL;
-            }
+            RebuildEntitySpatialLocation(TileCoordsXY{ x, y });
         }
     }
+    // Revuild next_in_quadrant linked list for LOCATION_NULL
```
Comment (ZehMatt) [inline src/openrct2/rct2/S6Exporter.cpp:1149]:
```suggestion
    // Rebuild next_in_quadrant linked list for LOCATION_NULL
```

File: distribution/changelog.txt [hunk 2]
```diff
@@ -16,6 +16,7 @@
 - Fix: [#14649] ImageImporter incorrectly remaps colours outside the RCT2 palette.
 - Fix: [#14667] “Extreme Hawaiian Island” has unpurchaseable land tiles (original bug).
 - Fix: [#15096] Crash when placing entrances in the scenario editor near the map corner.
+- Fix: [#15136] SV6 export causing vanilla RCT2 to crash under certain situations.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:19]: It was a freeze and not a crash. Also, I think it was in most or even all situations.

```suggestion
- Fix: [#15136] Exported SV6 files cause vanilla RCT2 to hang.
```

**Review by Gymnasiast — COMMENTED** [review]
I have a minor thingy about the changelog.

Apart from that, I have tested this PR and it works for me :+1: . There is another (unrelated) issue with map animations, but that’s something for its own issue/PR.

File: distribution/changelog.txt (hunk 2 above)
Comment (duncanspumpkin) [reply distribution/changelog.txt:19]: Hmm well that means i fixed two bugs then as it was crashing this morning after only the e16a4a0 commit.

**Review by Gymnasiast — APPROVED** [review]

**Review by ZehMatt — APPROVED** [review]
👍

**Pull request merged**